src/python/RucioUtils.py

```diff
@@ -14,12 +14,15 @@ def getNativeRucioClient(config=None, logger=None):
     logger.info("Initializing native Rucio client")
     from rucio.client import Client
 
+    rucio_cert = getattr(config.Services, "Rucio_cert", config.TaskWorker.cmscert)
+    rucio_key = getattr(config.Services, "Rucio_key", config.TaskWorker.cmskey)
+    logger.debug("rucio cert: %s\nrucio key: %s", rucio_cert, rucio_key)
     nativeClient = Client(
         rucio_host=config.Services.Rucio_host,
         auth_host=config.Services.Rucio_authUrl,
         ca_cert=config.Services.Rucio_caPath,
         account=config.Services.Rucio_account,
-        creds={"client_cert": config.TaskWorker.cmscert, "client_key": config.TaskWorker.cmskey},
```

Comment:
these looks like good changes, improve code quality, but maybe better not to mix RucioUtils in this PR ?

Reply:
Sorry, I forget to remove.
I need servicecert from crab-dev-tw01 in order to submit tape recall as `crab_tape_recall` rucio user.